packages/psss/src/api/queries/useAlerts.js

```diff
@@ -19,9 +20,14 @@ export const useAlerts = (period, orgUnitCodes, alertCategory) => {
     ...reportRow,
     syndromeName: SYNDROMES[reportRow.syndrome.toUpperCase()],
   }));
+  const sortedData = orderBy(
```

Comment:
This is great, it's such a clean way to get multi-sort.

Reply:
Yay for lodash!